roles/edpm_neutron_metadata/templates/wrappers/common_part.j2

```diff
@@ -1,7 +1,7 @@
 {% raw -%}
 
 CLI="nsenter --net=/run/netns/${NETNS} --preserve-credentials -m -t 1 podman"
-LOGGING="--log-driver k8s-file --log-opt path=/var/log/containers/stdouts/${NAME}.log"
+LOGGING="--log-driver journald"
```

Comment:
what exactly is this being used for.

is it still the hack for spawning metadata proxies in there own container that tripleo had.

you should not be using any scripts like this in general for managing the neutorn services.

i would strongly recommend considering running one metadata service in a single container managing multiple network namespaces for all tenenat.

the tripleo feature really does not strongly improve the security in a meaningful way and singnifcally complicate the management of the metadataproxy.

this so me is an existing defect in how this is deploy and is out of scope of this pr to change but i never thought that was a good idea in the tripleo days and i am sad to see this is still here in the new installer.

Reply:
do you mean the logging or the sidecars for Neutron?
the latter is something we cannot solve without native support in Neutron, so these wrappers to stay as an architectural tech debt. @booxter fyi

Reply:
yes i mean the sidecare contaienr shoudl be remvoed and we shoudl jsut run this they way everyone else does without them

the sidecares are just a triple feature.

we don't use container with upstream neutorn for example when runign with devstack

I'm 99% sure we don't do this in kolla-ansible or other installer either.

Reply:
let's put this off this patch discussion, there is a long story behind reevaluating sidecars for NextGen architecture